about.html

```diff
@@ -0,0 +1,170 @@
+          </div>
+
+          <div class="about_top">
+          <p class="about-paragraph">
+            Participant will get an opportunity to partcipate in various
+            pre-conferences, panel discussions, lectures, and hands-on
+            workshops, coverind a wide range of topics and disciplines.
+          </p>
+          <div class="about_contact">
+            <small>
+              please contact us via Email for any further questions about FF
+              National Summits!
+            </small>
+            <p class="text_mail">ffsummits2022@ffnaija.org</p>
+            <hr class="mail-line" />
+          </div>
+        </div>
+      </div>
+
+        <!-- Programs -->
+        <article class="about-section">
+          <div class="logo-text-div">
+            <h2 class="logo-text">Fit-Fam National Summits 2022 Logo</h2>
+            <hr class="about_hr" />
+            <small class="small_paragraph"
+              >Lorem ipsum dolor sit amet consectetur adipisicing elit. At
+              quisquam cupiditate nam animi veritatis eum. Molestias doloremque
+              asperiores dolorum iusto Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel, excepturi!</small
+            >
+          </div>
+
+          <div class="about_logo"><img src="./img/fitfam.jpg" alt="" /></div>
+          <hr />
+
+          <div class="logo-text-div header">
+            <h2 class="logo-text">See the past F-F National summits</h2>
+            <hr class="about_hr" />
+            <small class="small_paragraph"
+              >Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quis
+              quas rem placeat deleniti unde aut sapiente nostrum, laudantium et amet.
+              delectus.</small
+            >
+          </div>
+        </article>
+
+        <article class="picture_presentation">
+          <div class="previous_events">
+            <div class="event_image">
+              <img
+                src="./img/fit-women-conference-2021.jpg"
+                alt="past-event-img"
+              />
+              <span class="conference_year year">2015</span>
+              <span class="conference_venue venue"
+                >Dominion City Ajah Lagos Nigeria</span
+              >
+            </div>
+            <div class="event_image2">
+              <img
+                src="./img/fit-women-conference-2021.jpg"
+                alt="past-event-img"
+              />
+            </div>
+          </div>
+        </article>
+      </section>
+
+      <footer>
+        <div class="footer_bg">
+          <div class="program_h2 partner_h2">
+            <h2 class="main_program">Partner</h2>
+            <hr class="program_hr" />
+          </div>
+
+          <ul class="footer_list list">
+            <li><img src="./img/dinma.jpg" alt="" /></li>
+            <li><img src="./img/food-logo.jpg" alt="" /></li>
+            <li><img src="./img/fitfam.jpg" alt="" /></li>
+          </ul>
+        </div>
```

Comment:
- [ ] Kindly, remove the Partners from the About page, as this section is not present in the given design:

Given design:
![image](https://user-images.githubusercontent.com/83596135/163587714-d5d40ebb-1bab-4503-ab09-bf74773aa009.png)